doc/API.md

```diff
@@ -368,6 +369,38 @@ disregarded for forward compatibility.
 None; the `result` field will be set to the empty object `{}`. Any value should be
 disregarded for forward compatibility.
 
+### `gethistory`
+
+`gethistory` retrieves a paginated list of accounting events.
+
+The new deposit created by a change output is not included in the list
+of events.
+
+The `amount` of a `spend` event resource does not include the value of
+the change output.
+
+#### Request
+
+| Field    | Type         | Description                                                          |
+| -------- | ------------ | -------------------------------------------------------------------- |
+| `cursor` | int          | Timestamp of the date to retrieve events after                       |
+| `limit`  | int          | Number of events to retrieve                                         |
+| `kind`   | string array | Type of the events to retrieve, can be `deposit`, `cancel`, `spend`  |
+
+#### Response
+
+| Field          | Type   | Description                                |
+| -------------- | ------ | ------------------------------------------ |
+| `events`       | array  | Array of [Event resource](#event-resource) |
+
+##### Event Resource
+
+| Field    | Type   | Description                                                                              |
+| -------- | ------ | ---------------------------------------------------------------------------------------- |
+| `kind`   | string | Type of the event, can be `deposit`, `cancel`, `spend`                                   |
+| `date`   | int    | Timestamp of the event                                                                   |
+| `amount` | int    | Outgoing or incoming Amount of the event transaction, `0` if the event is a `cancel`     |
```

Comment:
May i suggest:
> Absolute amount, in sats, that is entering or exiting the wallet.

Which implies:
- that a Cancel, which "stays in the wallet" has an amount null (instead of special-casing it in the doc)
- that a Spend is obviously exiting
- that a Deposit is obviously entering